### PR TITLE
Hotfix/tests

### PR DESCRIPTION
**Pull request by vlazzarini**
Trying to enable csdtests failure.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -178,7 +178,7 @@ option(BUILD_UTILITIES "Build stand-alone executables for utilities that can als
 option(NEW_PARSER_DEBUG "Enable tracing of new parser" OFF)
 option(BUILD_MULTI_CORE "Enable building for multicore system" ON)
 option(FAIL_MISSING "Fail when a required external dependency is not present (useful for packagers)" OFF)
-option(USE_GETTEXT "Use the Gettext internationalization library" ON)
+option(USE_GETTEXT "Use the Gettext internationalization library" OFF)
```
Comment (kunstmusik) [inline CMakeLists.txt]: Not sure this should be set to OFF by default. It should gracefully not use gettext if it's not found.

**Comment by kunstmusik** [comment]
Note: This is now failing correctly and exposes that our macOS CI builds have problems running tests.

**Comment by vlazzarini** [comment]
yes, both brew and VCPKG CI have problems.

**Comment by vlazzarini** [comment]
The failures have to do with opcodes not being found. I managed to reproduce this locally and the solution seemed to be to fix the tests to find the opcodes correctly and to build plugins. But it is still failing.

**Comment by vlazzarini** [comment]
OK this seems to have been fixed now.

File: CMakeLists.txt (hunk 1 above)
Comment (kunstmusik) [reply CMakeLists.txt]: @vlazzarini can you revert this change for this PR as it doesn't have to do with fixing the tests? We can track it in the other PR.

Comment (vlazzarini) [reply CMakeLists.txt]: done

**Review by kunstmusik — APPROVED** [review]

**Pull request merged**